packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "@itwin/grouping-mapping-widget",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/grouping-mapping-widget.js",
+  "module": "lib/esm/grouping-mapping-widget.js",
+  "typings": "lib/cjs/grouping-mapping-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": "",
+    "test:watch": "npm -s test -- --reporter min --watch-extensions ts,tsx --watch"
+  },
+  "dependencies": {
+    "@itwin/imodel-browser-react": "^0.12.1",
+    "@itwin/itwinui-css": "^0.44.1",
+    "@itwin/itwinui-react": "^1.29.2",
+    "@itwin/itwinui-icons-react": "^1.5.1",
+    "classnames": "^2.3.1",
+    "cross-fetch": "^3.1.4",
+    "i18next": "^10.2.2",
+    "immer": "^9.0.7",
+    "react-spring": "^8.0.27",
+    "react-table": "^7.7.0",
```

Comment:
you were just using this to import the CellProps type right? can be a dev dep if so.
did you ever figure out why you needed to import this pkg for this type? wasnt in itwinui-react/wasnt re-exported there?

Reply:
I could not figure out why PNPM was not pulling in dependencies from itwinui-react. So I had to manually add it. If I'd do a regular npm install it would work.

Reply:
but should still be a dev dep no?

Reply:
Yup agreed.